examples/async_tasks/external_source_external_thread.rs

```diff
@@ -54,7 +54,7 @@ fn read_stream(receiver: Res<StreamReceiver>, mut events: EventWriter<StreamEven
 }
 
 fn spawn_text(mut commands: Commands, mut reader: EventReader<StreamEvent>) {
-    let text_style = TextStyle::default();
+    let text_style = TextFont::default();
```

Comment:
nit: rename this var

Reply:
Oh it can be be removed even, it's not needed with required components.